### DISCUSSION TITLE
Error in my first Symfony Project version 3.0

### DISCUSSION THREAD

**Discussion by ItsAnfield** [symfony/symfony#47679] [Q&A]
I may think its a composer error, but its installed without any issues

![image](https://user-images.githubusercontent.com/78814210/192094970-56411804-92ca-4fd1-b370-95d0e76544dd.png)

**Comment by ahundiak** [comment]
In general, using images to post code/errors is very much frowned upon.  Not just here but on most development boards.  I mean look at your question.  Can you read your image?  Even if you click on it (always a dangerous thing to do on the internet)  can you copy/paste stuff from it to use in a potential answer?  Consider editing your question.

You might also want to explain why you are trying to create a 3.0 project if that is indeed the version you are targeting.  I can maybe see a possible case for LTS 3.4 version if you have a bunch of legacy projects but even that is questionable.  It also helps to explain how you tried to create the project.  Perhaps even going so far as to reference which docs you were following.  Which, as a useful side effect, demonstrates that you did indeed read the docs.
